client/src/components/ChatSideBanner/ChatSideBanner.tsx

```diff
@@ -10,10 +10,11 @@ import AuthMenu from '../AuthMenu/AuthMenu';
 
 interface Props {
   loggedInUser: User;
+  onHandleOpen: () => void;
   handleDrawerToggle?: () => void;
 }
 
-const ChatSideBanner = ({ loggedInUser }: Props): JSX.Element => {
+const ChatSideBanner = ({ onHandleOpen, loggedInUser }: Props): JSX.Element => {
   const [search, setSearch] = useState<string>('test');
   const [newChatUser, setNewChatUser] = useState<User | null>(null);
   const classes = useStyles();
```

Comment:
Just a note, we actually aren't going to need this component at all.

Reply:
I would need the updated board ui, to build up on it :)